Release note for the CIE-5486 defect (#3802)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2025.0.6-offline-bug.md

---
date: 2024-01-07

> title: Offline installation blocked when Edge cannot access external DNS servers

Reviewer: ```suggestion
title: Offline installation no longer blocked when Edge cannot access external DNS servers
```

change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5486
version: 2025.0.6
---

> When installing Edge from an offline package, we discovered that installation was blocked when external DNS servers were unavailable. As Edge offline packages are meant to be used for installation and upgrade in an air-gapped environment, this behaviour is a defect and has now been fixed.

Reviewer: ```suggestion
When installing Edge from an offline package, the installation was blocked when external DNS servers were unavailable. As Edge offline packages are intended for use in installation and upgrade in an air-gapped environment, this behavior is a defect and has now been fixed.
```

Author: American spelling is our style guide, really? :-D
